src/components/Animal.js

```diff
@@ -16,6 +20,17 @@ const Animal = ({ animalsDb, onAddAnimal, onUpdateAnimal, onDeleteAnimal }) => {
     const [availability, setAvailability] = useState('')
     const [newsItem, setNewsItem] = useState('')
     const [description, setDescription] = useState('')
+    const objectMapping = {
+        "id": id,
+        "type": type,
+        "selectBreeds": selectBreeds,
```

Comment:
What is this selectBreeds key for?
I see that you also have a breeds key so was curious what the difference was between these two keys in this object

Reply:
selectBreeds is used to dynamically populate the breed choices based on selected type.